adafruit_veml6075.py

```diff
@@ -0,0 +1,194 @@
+    """
+
+    def __init__(self, i2c_bus, *, integration_time=50, high_dynamic=True,
+                 uva_a_coef=2.22, uva_b_coef=1.33,
+                 uvb_c_coef=2.95, uvb_d_coef=1.74,
+                 uva_response=0.001461, uvb_response=0.002591):
+        # Set coefficients
+        self._a = uva_a_coef
+        self._b = uva_b_coef
+        self._c = uvb_c_coef
+        self._d = uvb_d_coef
+        self._uvaresp = uva_response
+        self._uvbresp = uvb_response
+        self._uvacalc = self._uvbcalc = None
+
+        # Init I2C
+        self._i2c = I2CDevice(i2c_bus, _VEML6075_ADDR)
+        self._buffer = bytearray(3)
+
+        # read ID!
+        veml_id = self._read_register(_REV_ID)
+        if veml_id != 0x26:
+            raise RuntimeError("Incorrect VEML6075 ID 0x%02X" % veml_id)
+
+        # shut down
+        self._write_register(_REG_CONF, 0x01)
+
+        # Set integration time
+        self.integration_time = integration_time
+
+        # enable
+        conf = self._read_register(_REG_CONF)
+        if high_dynamic:
+            conf |= 0x08
+        conf &= ~0x01  # Power on
+        self._write_register(_REG_CONF, conf)
+
+    def _take_reading(self):
+        """Perform a full reading and calculation of all UV calibrated values"""
+        time.sleep(0.1)
+        uva = self._read_register(_REG_UVA)
+        uvb = self._read_register(_REG_UVB)
+        #dark = self._read_register(_REG_DARK)
+        uvcomp1 = self._read_register(_REG_UVCOMP1)
+        uvcomp2 = self._read_register(_REG_UVCOMP2)
+        # Equasion 1 & 2 in App note, without 'golden sample' calibration
+        self._uvacalc = uva - (self._a * uvcomp1) - (self._b * uvcomp2)
+        self._uvbcalc = uvb - (self._c * uvcomp1) - (self._d * uvcomp2)
+        #print("UVA = %d, UVB = %d, UVcomp1 = %d, UVcomp2 = %d, Dark = %d" %
+        #      (uva, uvb, uvcomp1, uvcomp2, dark))
+
+    @property
+    def uva(self):
+        """The calibrated UVA reading, in 'counts' over the sample period"""
+        self._take_reading()
+        return self._uvacalc
+
+    @property
+    def uvb(self):
+        """The calibrated UVB reading, in 'counts' over the sample period"""
+        self._take_reading()
+        return self._uvbcalc
+
+    @property
+    def uv_index(self):
+        """The calculated UV Index"""
+        self._take_reading()
+        return ((self._uvacalc * self._uvaresp) + (self._uvbcalc * self._uvbresp)) / 2
+
+    @property
+    def integration_time(self):
+        """The amount of time the VEML is sampling data for, in millis"""
+        key = (self._read_register(_REG_CONF) >> 4) & 0x7
+        for k, val in enumerate(_VEML6075_UV_IT):
+            if key == k:
+                return val
+        raise RuntimeError("Invalid integration time")
+
+    @integration_time.setter
+    def integration_time(self, val):
+        """Set how long the VEML samples data.
+        Can be 50, 100, 200, 400 or 800ms"""
```

Comment:
Most other libraries only have the docstring in the `@property` function. RTD _may_ render `integration_time` twice or combine them, which could be confusing. I would just add something like, "Valid times are: 50, 100, 200, 400, or 800" to the end of the `@property` docstring.

Although, after reviewing a couple other doc pages, maybe we should start/change the docstrings to include the setter functions. As they stand, the API documentation does not show the available args. This may be a good in-the-weeds topic for the meeting on Monday though. Thoughts @kattni @tannewt?